{
  "affected": [
    {
      "database_specific": {
        "source": "https://storage.googleapis.com/cve-osv-conversion/osv-output/CVE-2025-5918.json"
      },
      "ranges": [
        {
          "events": [
            {
              "introduced": "0"
            },
            {
              "fixed": "70ff28fcf04ec129a1d064f96e49aa57fcc90e37"
            }
          ],
          "repo": "https://github.com/libarchive/libarchive",
          "type": "GIT"
        }
      ],
      "versions": [
        "v2.6.0",
        "v2.6.1",
        "v2.6.2",
        "v2.7.0",
        "v2.7.1",
        "v2.8.0",
        "v2.8.1",
        "v2.8.2",
        "v2.8.3",
        "v2.8.4",
        "v2.8.5",
        "v3.0.0a",
        "v3.0.1b",
        "v3.0.2",
        "v3.0.3",
        "v3.0.4",
        "v3.1.0",
        "v3.1.1",
        "v3.1.2",
        "v3.1.900a",
        "v3.1.901a",
        "v3.2.0",
        "v3.2.1",
        "v3.2.2",
        "v3.3.0",
        "v3.3.1",
        "v3.3.2",
        "v3.3.3",
        "v3.4.0",
        "v3.4.1",
        "v3.4.2",
        "v3.4.3",
        "v3.5.0",
        "v3.5.1",
        "v3.5.2",
        "v3.6.0",
        "v3.6.1",
        "v3.6.2",
        "v3.7.0",
        "v3.7.1",
        "v3.7.2",
        "v3.7.3",
        "v3.7.4",
        "v3.7.5"
      ]
    }
  ],
  "details": "A vulnerability has been identified in the libarchive library. This flaw can be triggered when file streams are piped into bsdtar, potentially allowing for reading past the end of the file. This out-of-bounds read can lead to unintended consequences, including unpredictable program behavior, memory corruption, or a denial-of-service condition.",
  "id": "CVE-2025-5918",
  "modified": "2025-10-16T05:55:08.774877Z",
  "published": "2025-06-09T20:15:27Z",
  "references": [
    {
      "type": "ADVISORY",
      "url": "https://access.redhat.com/security/cve/CVE-2025-5918"
    },
    {
      "type": "ADVISORY",
      "url": "https://github.com/libarchive/libarchive/releases/tag/v3.8.0"
    },
    {
      "type": "FIX",
      "url": "https://github.com/libarchive/libarchive/pull/2584"
    },
    {
      "type": "REPORT",
      "url": "https://bugzilla.redhat.com/show_bug.cgi?id=2370877"
    }
  ],
  "related": [
    "SUSE-SU-2025:02566-1",
    "SUSE-SU-2025:02718-1"
  ],
  "schema_version": "1.7.3",
  "severity": [
    {
      "score": "CVSS:3.1/AV:L/AC:L/PR:L/UI:R/S:U/C:H/I:N/A:H",
      "type": "CVSS_V3"
    }
  ]
}